python/cudf/cudf/core/buffer/cow_buffer.py

```diff
@@ -14,16 +14,16 @@
 T = TypeVar("T", bound="CopyOnWriteBuffer")
 
 
-def _keys_cleanup(ptr, size):
-    weak_set_values = CopyOnWriteBuffer._instances[(ptr, size)]
+def _keys_cleanup(ptr):
+    weak_set_values = CopyOnWriteBuffer._instances[(ptr,)]
     if (
         len(weak_set_values) == 1
         and next(iter(weak_set_values.data))() is None
     ):
         # When the last remaining reference is being cleaned up we will still
         # have a dead weak-reference in `weak_set_values`, if that is the case
         # we are good to perform the key's cleanup
-        del CopyOnWriteBuffer._instances[(ptr, size)]
+        del CopyOnWriteBuffer._instances[(ptr,)]
```

Comment:
Could just use `ptr` rather than `(ptr,)` now?

Reply:
Yup, we can use just `ptr` now. Made this change 👍